src/utils/userSchema.js

```diff
@@ -23,5 +23,9 @@ export default {
     lastName,
     email,
     password
+  }),
+  signin: Joi.object().keys({
+    email: Joi.string().required().label('Email is required'),
```

Comment:
I think it's necessary to validate this email, a user may input a number or a 2 letter word, we don't have to check the database to see if it exists. Let's use the schema provided above for that.
```suggestion
    email,
```